distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -264,4 +432,63 @@ private void showServerReply(FTPClient ftpClient) {
             }
         }
     }
-}
+
+    /**
+     * Sets the clientAuth configuration in catalog-ftp feature.
+     *
+     * @throws Exception
+     */
+    private void setClientAuthConfiguration(String clientAuth) throws Exception {
+        Configuration config =
+                getAdminConfig().getConfiguration("ddf.catalog.ftp.FtpServerStarter");
+        config.setBundleLocation("mvn:ddf.catalog/ftp/" + System.getProperty("ddf.version"));
+        Dictionary properties = new Hashtable<>();
```

Comment:
`config.getProperties()` is null before it is updated. Unless there is another way to get the original port and clientAuth, making a brand new table each time (like in FtpServerStarterTest.java) might be the best solution.